genestack_client/cla.py

```diff
@@ -161,14 +161,16 @@ def replace_file_reference(self, accession, key, accession_to_remove, accession_
         If ``accession_to_remove`` or ``accession_to_add`` is not found,
         an exception will be thrown.
 
-        :param accession: accession of file
+        :param accession: file accession or accession list
         :param key: key for source files
         :param accession_to_remove: accession to remove
         :param accession_to_add: accession to add
         :return: None
         """
-        self.invoke('replaceFileReference', accession, key, accession_to_remove, accession_to_add)
+        self.invoke('replaceFileReference', self.__accession_list__(accession), key, accession_to_remove, accession_to_add)
 
+    def __accession_list__(self, accession):
```

Comment:
Double underscore is syntax for special methods in python. This mothod is private should be named as `__accession_list`